src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderNoPayPeriod.java

```diff
@@ -1,19 +1,21 @@
 package uk.gov.justice.hmpps.prison.repository.jpa.model;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
 import java.time.LocalDate;
 
 @Data
-@Builder
+@SuperBuilder
```

Comment:
Needed to add this in a few places so that the audit fields are included with the Lombok builder meaning we can satisfy the `NOT NULL` audit columns.